app/serializers/api/admin/enterprise_serializer.rb

```diff
@@ -15,7 +15,7 @@ class EnterpriseSerializer < ActiveModel::Serializer
                  :terms_and_conditions_file_name, :terms_and_conditions_updated_at,
                  :preferred_invoice_order_by_supplier, :preferred_product_low_stock_display,
                  :visible, :hide_ofn_navigation, :white_label_logo,
-                 :white_label_logo_link
+                 :white_label_logo_link, :hide_groups_tab
```

Comment:
I don't see this used anywhere. Do we need to serialise it?

Reply:
We don't need it. Removed!